src/fermilib/utils/_operator_utils.py

```diff
@@ -102,3 +111,102 @@ def commutator(operator_a, operator_b):
             isinstance(operator_b, (QubitOperator, FermionOperator))):
         return operator_a * operator_b - operator_b * operator_a
     raise TypeError('Operator of invalid type.')
+
+
+def save_operator(operator, file_name=None, data_directory=None):
+    """Save FermionOperator or QubitOperator to hdf5 file.
+
+    Args:
+        operator: An instance of FermionOperator or QubitOperator.
+        file_name: The name of the saved hdf5 file. If not provided, the unix
+            time when the saving happens will be used.
+        data_directory: Optional data directory to change from default data
+            directory specified in config file.
+
+    Raises:
+        OperatorUtilsError: Not saved, file already exists.
+        TypeError: Operator of invalid type.
+    """
+    file_path = get_file_path(file_name, data_directory)
+
+    if os.path.isfile(file_path):
+        raise OperatorUtilsError("Not saved, file already exists.")
+
+    if isinstance(operator, FermionOperator):
+        operator_type = "FermionOperator"
+    elif isinstance(operator, QubitOperator):
+        operator_type = "QubitOperator"
+    else:
+        raise TypeError('Operator of invalid type.')
+
+    with h5py.File(file_path, 'w') as f:
+        f['operator_type'] = numpy.string_(operator_type)
+        f['operator_terms'] = numpy.string_(json.dumps(
+            [{'k': k, 'r': v.real, 'i': v.imag}
+                for k, v in operator.terms.iteritems()],
+            ensure_ascii=False).encode('utf8'))
+
+
+def load_operator(file_name=None, data_directory=None):
+    """Load FermionOperator or QubitOperator from hdf5 file.
+
+    Args:
+        file_name: The name of the saved hdf5 file. If not provided, the unix
```

Comment:
I'm a little confused about what the default file name is when loading. I feel like the filename should not be optional for the load function. Do you agree?